lib/vagrant-service-manager/command.rb

```diff
@@ -102,12 +111,20 @@ def print_help(config = {})
       end
 
       def check_if_a_service_is_running?(service)
+        return check_if_kube_is_running? if ['kubernetes', 'k8s'].include? service
```

Comment:
You may want to extract `['kubernetes', 'k8s']`  as `KUBE_NAMES` or any preferred named constant in plugin level.